File: JS/02.Basics/02.2-ConditionalStatementsLAB/07.AreaofFigures.js
function calculateArea(figure, param1, param2) {
    let area = 0;

    switch (figure) {
        case "square":
            area = param1 * param1;
            break;
        case "rectangle":
            area = param1 * param2;
            break;
        case "circle":
            area = Math.PI * param1 * param1;
            break;
        case "triangle":
            area = (param1 * param2) / 2;
            break;
    }

    return area.toFixed(3);
}

console.log(calculateArea("square", 5));
console.log(calculateArea("rectangle", 7, 2.5));
console.log(calculateArea("circle", 6));
console.log(calculateArea("triangle", 4.5, 20));

File: C++/01-Basics/02.2-ConditionalStatementsLAB/07.AreaofFigures/07.AreaofFigures.cpp
#include <iostream>
#include <cmath>
#include <iomanip>

using namespace std;

int main() {
    double pi = 3.14159265359;

    string figure;
    cin >> figure;

    cout << fixed << setprecision(3);

    if (figure == "square") {
        double side;
        cin >> side;
        cout << side * side << endl;
    }
    else if (figure == "rectangle") {
        double width, height;
        cin >> width >> height;
        cout << width * height << endl;
    }
    else if (figure == "circle") {
        double radius;
        cin >> radius;
        cout << pi * radius * radius << endl;
    }
    else if (figure == "triangle") {
        double side, height;
        cin >> side >> height;
        cout << (side * height) / 2 << endl;
    }

    return 0;
}
